### PR TITLE
Remove Tools::addonsRequest('module') calls

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Remove Tools::addonsRequest('module') calls
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes PrestaShop/Prestashop#24750
| How to test?  | N/A

### CODE REVIEW

**Review by atomiix — APPROVED** [review]

File: controllers/admin/AdminPsThemeCustoConfiguration.php [hunk 1]
```diff
@@ -527,16 +527,7 @@ public function setFinalList($aList)
                 } else {
                     foreach ($aElementsList as $sModuleName => $iModuleId) {
                         if (!in_array($sModuleName, $modulesOnDisk)) {
-                            if ($this->getModule()->ready !== false) {
```
Comment (PierreRambaud) [inline controllers/admin/AdminPsThemeCustoConfiguration.php:530]: `$this->getModule()->ready` what the fuck is that?

Could we add a hook at least? In order to be able to redownload or do something if the module is not on the disk?

**Review by PierreRambaud — COMMENTED** [review]
Simple comment

File: controllers/admin/AdminPsThemeCustoConfiguration.php (hunk 1 above)
Comment (Progi1984) [reply controllers/admin/AdminPsThemeCustoConfiguration.php:530]: `$this->ready = (getenv('PLATEFORM') === 'PSREADY') ? true : false;` Oh shit !

Comment (Progi1984) [reply controllers/admin/AdminPsThemeCustoConfiguration.php:530]: @PierreRambaud Done in #49 49

Comment (PierreRambaud) [reply controllers/admin/AdminPsThemeCustoConfiguration.php:530]: You forget to read my second question little padawan :trollface:

Comment (Progi1984) [reply controllers/admin/AdminPsThemeCustoConfiguration.php]: It's not on the scope of the PR. It should be done when an API will be available.

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**